decidim-core/app/models/decidim/authorization.rb

```diff
@@ -31,14 +31,32 @@ def granted?
       !granted_at.nil?
     end
 
+    # Calculates at when this authorization will expire, if it needs to.
+    #
+    # Returns nil if the authorization does not expire.
+    # Returns an ActiveSupport::TimeWithZone if it expires.
+    def expires_at
+      return unless workflow_manifest
+      return if workflow_manifest.expires_in.zero?
+      granted_at + workflow_manifest.expires_in
+    end
+
+    def expired?
+      expires_at.present? && expires_at < Time.zone.now
+    end
+
     private
 
     def active_handler?
-      if Decidim::Verifications.find_workflow_manifest(name)
+      if workflow_manifest.present?
```

Comment:
You could just skip the whole `if` statement and simply do `workflow_manifest.present?` here xD.

Reply:
LOOOOOOOOOOOOOOL indeed